docs/string/trie.md

```diff
@@ -1,36 +1,36 @@
-字典树，英文名 Trie。顾名思义，就是一个像字典一样的树。
+Trie, also called prefix tree or digital tree, is a tree like a dictionary.
 
-## 简介
+## Introduction
 
-先放一张图：
+Let's look at a figure first:
 
 ![trie1](./images/trie1.png)
 
-可以发现，这棵字典树用边来代表字母，而从根结点到树上某一结点的路径就代表了一个字符串。举个例子， $1\to4\to 8\to 12$ 表示的就是字符串 `caa` 。
+It can be found that this trie uses edges to represent letters, and the path from the root node to a node on the tree represents a string. For example, $1\to4\to 8\to 12$ represents the string `caa`.
 
-Trie 的结构非常好懂，我们用 $\delta(u,c)$ 表示结点 $u$ 的 $c$ 字符指向的下一个结点，或着说是结点 $u$ 代表的字符串后面添加一个字符 $c$ 形成的字符串的结点。（ $c$ 的取值范围和字符集大小有关，不一定是 $0\sim 26$ 。）
+The structure of trie is very easy to understand. We use $\delta(u,c)$ to represent the next node pointed to by the $c$ character of the node $u$ , or it can be seen as adding a node of the string formed by the character $c$ after the string represented by the node $u$ . (The value range of $c$ is related to the size of the character set. It is not necessarily $0\sim 26$ .)
```

Comment:
>or it can be seen as adding a node of the string ...
or it can be seen as the node representing the string which is formed by appending an character $c$ after the string represented by node $u$